app/Dockerfile

```diff
@@ -7,8 +7,8 @@ RUN apk -U upgrade
 
 
 # setup user stuff
-ARG RUN_UID
-ARG RUN_USER
+ARG RUN_UID=${UID:-4000}
+ARG RUN_USER=${USER:-'nodummy'}
```

Comment:
I couldn't find any documentation that this syntax is supported here. Does it work?